user-guide/Advanced_Functionality/Swarming/SwarmingBookings.md

```diff
@@ -0,0 +1,102 @@
+---
+uid: SwarmingBookings
+---
+
+# Swarming bookings
+
+Since 10.4.4, it is possible to swarm bookings. To do so, you need the *Swarming* soft-launch option. 
+From 10.5.1 onwards, the feature is placed behind a different soft-launch option, namely *BookingSwarming*.
+
+Swarming bookings can be done by sending a *SwarmingRequestMessage*, to which you add the new hosting agent ID and booking ID. 
```

Comment:
Maybe clarify that swarming a booking means that the event scripts of the booking will be executed on the new hosting agent. This doesn't seem to be common knowledge.

Reply:
Maybe add that internally we will first try to unregister the booking from the old agent and wait for any running actions, then register the booking on the new hosting agent.